### ISSUE TITLE
MIDI In

### ISSUE THREAD

**Issue by ayskura** [0xtob/nitrotracker#63]
I would like to use it with midi in, to play sample from a midi keyboard. I saw in the internet about the DSerial but all pages are gone and I can't find any resources... would it be possible to have midi input on Nitrotracker?

**Comment by deadpahn** [comment]
There is a weird [DSWIFI MIDI ](https://www.gamebrew.org/wiki/DSMI_Server)thing that I think 4 people in the world used for 13 minutes at some point. Over the air seems to be the only way. There are some [insane projects](https://makezine.com/article/home/fun-games/nintendo-ds-midi/) but I wouldnt hold your breath.

I really don't see the point of having it. Local DS connectivity (to have more than 1 instance of NT working in sync) would be much better/useful use of time.

**Comment by ayskura** [comment]
the point of having a midi input is to be able to play it as a sampler... it's so powerful for such a small battery powered device.... I really look for something like this, a tracker portable with midi input. Polyend Mini is overpriced, and there aren't really alternatives...

**Comment by deadpahn** [comment]
I agree with the sampler part. I'm not sure what the DACs are on the DS but
it sounds like samplers from the golden era. People are really sleeping on
making music on the NDS with samples, it's like having an S900 in your
backpack.

**Comment by ayskura** [comment]
exactly! and with a low quality microphone, and the possibility to edit directly the samples with the pen... it's really something special! especially trying to make the DS boot directly into Nitrotracker it's such an amazing tool... All the docs relative to the midi input are not on internet anymore, most links are gone... but someone did it before so it's possible to do it - it would be awesome!

**Comment by gbrendel** [comment]
Not something I'd use much but the novelty would surely be fun!

**Comment by illettante** [comment]
> especially trying to make the DS boot directly into Nitrotracker
You might already know but on DSi you can use Unlaunch to boot directly to Nitro, and use power button + a/b/x/y to boot to stock system or firmware/app of your choice. not sure of DS equivalent but there will be a way!
